rfc/0054-sdk3-management-apis.md

```diff
@@ -582,6 +666,40 @@ void BuildDeferredIndexes(string bucketName, [options])
 
   * `Timeout` or `timeoutMillis` (`int`/`duration`) - the time allowed for the operation to be terminated. This is controlled by the client.
 
+  * `CollectionName` - the name of the collection to restrict indexes to.
+  * `ScopeName` - the name of the scope to restrict indexes to.
+    * If either `CollectionName` or `ScopeName` are set then both *must* be set.
+
+### N1QL
+
+This is done by performing 2 querys; one to fetch the deferred indexes to build and the other to build the deferred
+indexes.
+
+No collection set:
```

Comment:
To handle the case where the user explicitly specifies the default collection, should this be:

    No collection set, or collection set to "_default":

Then down below, should the condition for the alternate path be:

    Collection set (and not "_default")